engine/src/flutter/shell/platform/windows/task_runner_window.cc

```diff
@@ -69,6 +74,16 @@ void TaskRunnerWindow::RemoveDelegate(Delegate* delegate) {
   }
 }
 
+void TaskRunnerWindow::PollOnce() {
```

Comment:
What do you think of accepting a timeout argument? That would make it clearer that this method blocks if the event loop is empty. `FlutterWindowsView::OnWindowSizeChanged` could pass its window resize timeout as that argument.

Reply:
I like it. Added.